packages/table/src/cell/editableCell.tsx

```diff
@@ -96,9 +96,7 @@ export class EditableCell extends React.Component<IEditableCellProps, IEditableC
 
     public componentWillReceiveProps(nextProps: IEditableCellProps) {
         const { value } = nextProps;
-        if (value !== this.props.value) {
```

Comment:
@maclockard i believe removing this check is wrong. it should instead by `!= null`, [just like `EditableText` does it](https://github.com/palantir/blueprint/blob/develop/packages/core/src/components/editable-text/editableText.tsx#L206-L208). i'm going to close this PR and open my own with the correct change, and a refactor to the corresponding test.

thanks for suggesting this feature!